src/axom/mir/docs/sphinx/mir_algorithms.rst

```diff
@@ -0,0 +1,153 @@
+.. ## Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+******************************************************
+MIR Algorithms
+******************************************************
+
+The MIR component contains MIR algorithms that will take a Blueprint mesh as input,
+perform MIR on it, and output a new Blueprint mesh with the reconstructed output.
+A Blueprint mesh is contained in a ``conduit::Node`` and it follows the `Blueprint protocol <https://llnl-conduit.readthedocs.io/en/latest/blueprint_mesh.html>`_,
+which means the node contains specific items that describe the mesh coordinates, topology, fields, and materials.
+
+#######
+Inputs
+#######
+
+MIR algorithms are designed to accept a Conduit node containing various options that can
+influence how the algorithm operates. The MIR algorithm copies the options node to the memory
+space where it will be used.
```

Comment:
Add something like "The only required option is `matset`, the name of the matset to operate on.  Other options have sensible defaults described in the table below."

Reply:
Added.